x/rewards/keeper/rewards.go

```diff
@@ -55,9 +55,9 @@ func (k Keeper) RefillRewardsPools(ctx sdk.Context, _ []byte, data []byte) {
 	// get the months left for the allocation pools
 	var monthsLeft uint64
 	if len(data) == 0 {
-		monthsLeft = uint64(types.RewardsAllocationPoolsLifetime)
+		monthsLeft = types.RewardsAllocationPoolsLifetime
 	} else {
-		monthsLeft = binary.BigEndian.Uint64(data)
+		monthsLeft = binary.BigEndian.Uint64(data) // TODO: should we check data or trust binary package?
```

Comment:
Are you going to resolve this TODO?

Reply:
I checked it and forgot to delete. fixed https://github.com/lavanet/lava/commit/47cf05ecbab2e476059ef74e97c8c31f867fc287